packages/protocol/src/interaction/EventDataDecoder.ts

```diff
@@ -33,24 +35,39 @@ export type DecodedEventData<T> = {
     data?: T;
 };
 
-export type DecodedEventReportValue<T> = {
+type DecodedEventReportEntry = {
```

Comment:
Hmm maybe should export this since we export types derived from it, I think it can cause definition file issues if you don't.